hat/assets/js/apps/Iaso/domains/app/components/SidebarMenuComponent.js

```diff
@@ -79,10 +80,13 @@ const styles = theme => ({
 });
 
 const localizedManualUrl = (locale, account) => {
-    if (locale === 'fr' && account === 'RDC') {
+    if (locale === 'fr' && account.name === 'RDC') {
```

Comment:
Why do we have this specific case ? I know you didn't write it in the first place but I'm just wondering. Shouldn't we have a way to have specific user manuals for specific locales. Like multiple user manuals per account (one for fr, one for en .. etc ?)

Reply:
I didn't change that code to avoid that persons who use IASO RDC don't miss the user guide. And for multiple languages for user manual , on my opinion it should be treated later.